src/main/java/com/monkeyrizz/codeforest/service/ProblemService.java

```diff
@@ -13,8 +13,16 @@ public List<Problem> getProblems() {
         return List.of(
                 new Problem(
                         "twosum",
-                        "return the two numbers that add up to a sum"
+                        "return the two numbers that add up to a sum",
+                        2L
                 )
         );
     }
+    public Problem getProblemById(int id) {
+        return new Problem(
+                "problem by ID",
+                "problem description by ID: " + String.valueOf(id),
+                3L
+        );

```

Comment:
This is also not a great way to test your code. You should implement unit tests to invoke your service methods. 

Since we don't have connection to live DB yet, we can use Mockito to write unit tests and mock the DB responses. We can discuss this further if necessary.